### ISSUE TITLE
Example in README doesn't work

### ISSUE THREAD

**Issue by txdv** [ysbaddaden/artanis#14]
I'm using crystal 0.17 on a debian 32 bit system and I am getting:

```
$ crystal run example.cr 
Error in ./example.cr:26: instantiating 'App:Class#call(HTTP::Server::Context)'

  App.call(context)
      ^~~~

in ./libs/artanis/application.cr:50: instantiating 'App#call()'

      new(context).call
                   ^~~~

in macro 'call_action' /media/disk/bentkus/Projects/crystal/artanis/libs/artanis/dsl.cr:87, line 1:

  1.       if __temp_99 = URI.unescape(request.path || "/").match(MATCH_GET__SLASH_)
  2.         __temp_100 = match_GET__SLASH_(__temp_99)
  3. 
  4.         
  5.           break unless __temp_100 == :pass
  6.         
  7.       end
  8.     

      if __temp_99 = URI.unescape(request.path || "/").match(MATCH_GET__SLASH_)
                                                             ^~~~~~~~~~~~~~~~~

undefined constant MATCH_GET__SLASH_
```

I haven't looked into the code yet, it was pointed out to me in the channel that this library makes use of the macro features and that I should look into it to see what is possible.

Any quick idea on why this error is thrown?

**Comment by jhass** [comment]
Using artanis master branch I get

```
Error in ./test.cr:26: instantiating 'App:Class#call(HTTP::Server::Context)'

  App.call(context)
      ^~~~

in ./libs/artanis/application.cr:50: instantiating 'App#call()'

      new(context).call
                   ^~~~

in ./libs/artanis/dsl.cr:138: expanding macro

      {% if methods.empty? %}
      ^

in ./libs/artanis/dsl.cr:138: undefined macro variable 'methods'

      {% if methods.empty? %}
            ^~~~~~~
```

for both 0.17 and 0.17.1.

**Comment by ysbaddaden** [comment]
This should have been fixed by #15.
